datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx

```diff
@@ -46,6 +48,13 @@ export const addFixedQuery = (baseQuery: string, fixedQuery: string, emptyQuery:
     return finalQuery;
 };
 
+// Simply remove the fields that were marked as fixed from the facets that the server
+// responds.
+export const removeFixedFiltersFromFacets = (fixedFilters: FilterSet, facets: FacetMetadata[]) => {
+    const fixedFields = fixedFilters.filters.map((filter) => filter.field);
+    return facets.filter((facet) => !(fixedFields.indexOf(facet.field) > -1));
```

Comment:
definitely a nit and not expecting you to change this - but could use `.includes` here as I think that's more readable (`facets.filter(facet => !fixedFields.includes(facet.field))`

Reply:
thank you!